.changeset/grumpy-worms-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': major
```

Comment:
Are you sure you want to release a new major release?

Reply:
True, changed to minor change